sp_mock/frontend/src/views/Home.vue

```diff
@@ -17,42 +16,26 @@ const userName = computed(() => {
     return "";
 });
 
-console.log("payload", JSON.parse(atob(token.value.split(".")[1])));
+//console.log("payload", JSON.parse(atob(token.value.split(".")[1])));
 
-const getPoems = async () => {
-    // try {
-    //   const poem = await layer8.getPoem({
-    //     method: "GET",
-    //     headers: {
-    //       "Content-Type": "application/json",
-    //       "Authorization": `${token.value}`,
-    //     }
-    //   });
-    //   if (poem) {
-    //     showPoem(poem);
-    //     gotAPoem.value = true;
-    //   } else {
-    //     console.error("No poem content received.");
-    //   }
-    // } catch (error) {
-    //   console.error(error);
-    //   alert("Poems failed!");
-    // }
-
-    // Since poems are not implemented in the new mock, just using a placeholder here
-    const poem =
-        "Roses are red, violets are blue, I'm a placeholder, and so are you.";
-    showPoem(poem);
-    gotAPoem.value = true;
+const getPoem = async () => {
+    try {
+        console.log("going to try now 1...")   
+        const resp = await layer8.fetch("http://localhost:8000/nextpoem");
```

Comment:
Wow you also implemented the getpoem functionality, which is going to transmit poem data through the tunnel, great work!